### PR TITLE
I updated the Services Section with key events: CyberStart America; W…

### LINKED ISSUE #2: Features
image

Using the old website, transfer some key events that the club provides. Try uploading images from our website to replace them as well.

### PR DESCRIPTION
**Pull request by cqsun2**
…eekly Challenges; Videos, Tutorials, and Resources; and Holiday Hackers CTF. I also added the descriptions for the old website. I haven't added the links associated with those or updated the icon pictures, since I don't know how to do that yet. I also replaced the Why Us Section video with a video of our logo.

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -133,32 +133,37 @@ <h2 class="animate__animated animate__fadeInDown">Sequi ea ut et est quaerat</h2
           <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up">
             <div class="icon-box icon-box-pink">
               <div class="icon"><i class="bx bxl-dribbble"></i></div>
-              <h4 class="title"><a href="">Lorem Ipsum</a></h4>
-              <p class="description">Voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident</p>
+              <h4 class="title"><a href="">CyberStart America</a></h4>
+              <h6 class="subtitle">Participate in this amazing program!</p>
+              <p class="description">For beginners to seasoned professionals, Cyberstart America  provides participants with engaging challenges, a game, and a National Championship! This is an amazing opportunity, and to learn more, click on the link below.</p>
+              <div href="link"><a class="btn-get-started animate__animated animate__fadeInUp">CyberStart America Website</a></div>
```
Comment (emsesc) [inline pages/index.html]: A lot of HTML/CSS is about experimenting - try this code, and be sure to change the "insert link...." part! Random tip: usually "hrefs" are what you use to create links.
```html
              <div class="carousel-item active"><a href="[insert link to CyberStart website]" class="btn-get-started animate__animated animate__fadeInUp">CyberStart America Website</a></div>
```

File: pages/index.html [hunk 2]
```diff
@@ -133,32 +133,37 @@ <h2 class="animate__animated animate__fadeInDown">Sequi ea ut et est quaerat</h2
...
           </div>
 
           <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="200">
             <div class="icon-box icon-box-green">
               <div class="icon"><i class="bx bx-tachometer"></i></div>
-              <h4 class="title"><a href="">Magni Dolores</a></h4>
-              <p class="description">Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</p>
+              <h4 class="title"><a href="">Videos, Tutorials, and Resources</a></h4>
+              <h6 class="subtitle">Obtain your ethical hacking skills!</p>
+              <p class="description">Let’s not wait until the next intake of GGCyberstart- the learning starts now. Check out our tutorials, resources, and videos, which are updated weekly, to dive deeper into the world of cybersecurity! (These are a supplement to the basics you learn during meetings).</p>
```
Comment (emsesc) [inline pages/index.html]: Could you update this description to say CyberStart America?

File: pages/index.html [hunk 3]
```diff
@@ -133,32 +133,37 @@ <h2 class="animate__animated animate__fadeInDown">Sequi ea ut et est quaerat</h2
...
           </div>
 
           <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="100">
             <div class="icon-box icon-box-cyan">
               <div class="icon"><i class="bx bx-file"></i></div>
-              <h4 class="title"><a href="">Sed ut perspiciatis</a></h4>
-              <p class="description">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur</p>
+              <h4 class="title"><a href="">Weekly Challenges</a></h4>
+              <h6 class="subtitle">Practice your secret agent skills!</p>
+              <p class="description">Come back and visit this website every Monday for a new challenge of the week! When you find the flag, submit it on our website. The first person to enter the correct flag will be featured on our Discord and website!</p>
```
Comment (emsesc) [inline pages/index.html:149]: Another random tip: if you want to create a bigger space between titles and descriptions, you can use the "break" HTML tag:
```html
<br></br>
```

**Review by emsesc — CHANGES_REQUESTED** [review]
@cqsun2 Great work!! Hopefully you're more familiar with HTML (and how annoying it can be) now. Just keep committing to this repositories to implement the changes I requested. LMK if you have questions :)

**Comment by emsesc** [comment]
@cqsun2 Does the button work now? You can attach a screenshot in a comment in this PR. Just make sure to remove the brackets in the `href` links - otherwise looks good!

**Comment by cqsun2** [comment]
@emsesc It still doesn't appear as a button, but it works as a link now that I have removed the brackets. 
Button

**Review by emsesc — APPROVED** [review]
We'll fix the button styling later, thanks @cqsun2!

**Pull request merged**